en/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md

```diff
@@ -13,16 +13,20 @@ This vulnerability has been assigned the CVE identifier [CVE-2023-28755](https:/
 
 ## Details
 
-A ReDoS issue was discovered in the URI component through 0.12.0 in Ruby through 3.2.1. The URI parser mishandles invalid URLs that have specific characters. It causes an increase in execution time for parsing strings to URI objects.
+A ReDoS issue was discovered in the URI component. The URI parser mishandles invalid URLs that have specific characters. It causes an increase in execution time for parsing strings to URI objects.
 
-Please update the uri gem to version 0.12.1 or later. We also release for old uri gem with Ruby releases. Please use them if you need to only security fix.
```

Comment:
The key part of this patch is to fix the following sentence where hard to understand:

> We also release for old uri gem with Ruby releases. Please use them if you need to only security fix.

I need to confirm with the original author to confirm the intention.